src/assets/stylesheets/Instructions.scss

```diff
@@ -269,3 +275,44 @@
     margin: 0;
   }
 }
+
+#app,
+#wc {
+  .c-instruction-tabs {
```

Comment:
Very nitpicky from me sorry, but wondering why you decided to go with the `c-` classname rather than matching the classnames we've already got in the instructions panel?

Reply:
Just because it is a component hence the `.c-` prefix, something I saw further up the file. Can remove but I thought this might be more consistent